stores/bench_test.go

```diff
@@ -0,0 +1,168 @@
+package stores
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"go.sia.tech/core/types"
+	isql "go.sia.tech/renterd/internal/sql"
+	"go.sia.tech/renterd/object"
+	"go.sia.tech/renterd/stores/sql"
+	"go.sia.tech/renterd/stores/sql/sqlite"
+	"go.uber.org/zap"
+	"lukechampine.com/frand"
+)
+
+const (
```

Comment:
Last nit. I'd inline all the consts and helpers into the benchmark as long as we don't need it anywhere else. To avoid adding arbitrary constants and very specific setup code to the general test scope